packages/devtools_app/lib/src/screens/deep_link_validation/project_root_selection/select_project_view.dart

```diff
@@ -59,25 +60,47 @@ class _SelectProjectViewState extends State<SelectProjectView>
     });
   }
 
-  void _handleValidateProject(String directory) async {
-    setState(() {
-      _retrievingFlutterProject = true;
-    });
+  Future<List<String>> _requestAndridVariants(String directory) async {
     ga.timeStart(gac.deeplink, gac.AnalyzeFlutterProject.loadVariants.name);
     final List<String> androidVariants =
         await server.requestAndroidBuildVariants(directory);
-    if (!mounted) {
+    if (androidVariants.isEmpty || !mounted) {
```

Comment:
is it possible that `androidVariants` could be empty even if this is a Flutter project?